### PR TITLE
feat/suggestion auto bootstrap namespace

### PR DESCRIPTION
**Pull request by GustavEikaas**
- add option to bootstrap filescoped namespace
- feat: suggestion

### CODE REVIEW

File: lua/easy-dotnet/options.lua [hunk 1]
```diff
@@ -138,16 +138,29 @@ local M = {
...
 
 local function merge_tables(default_options, user_options)
   return vim.tbl_deep_extend("keep", user_options, default_options)
 end
 
+--- Auto_bootstrap namespace can be either true or table with config
+local function handle_auto_bootstrap_namespace(a)
+  if type(a.auto_bootstrap_namespace) ~= "table" then
+    a.auto_bootstrap_namespace = {
+      type = "file_scoped",
```
Comment (cbosvik) [inline lua/easy-dotnet/options.lua:156]: should probably keep "block_scoped" as default as it was the only option before?

**Review by cbosvik — COMMENTED** [review]
nice! but probably best to keep block_scoped as default

File: lua/easy-dotnet/options.lua (hunk 1 above)
Comment (GustavEikaas) [reply lua/easy-dotnet/options.lua:156]: Agreed

**Comment by cbosvik** [comment]
Hey! I just found your plugin today, very happy I did. You have done some amazing work! Thank you!

**Comment by GustavEikaas** [comment]
Awesome! Glad you like it! ❤

**Comment by GustavEikaas** [comment]
@cbosvik If this looks good do you want to sync these changes into your PR? You are the author of the PR so it would be nice to merge your PR

**Pull request closed without merging**